src/site/content/en/blog/referrer-best-practices/index.md

```diff
@@ -285,34 +278,40 @@ app.use(helmet.referrerPolicy({policy: 'strict-origin-when-cross-origin'}));
 
 ### What if `strict-origin-when-cross-origin` (or stricter) doesn't accommodate all your use cases?
 
-In this case, don't set an unsafe policy such as `unsafe-url`. What you can do instead is take a
-**progressive approach**: set a protective policy for your website and if need be, a more permissive
-policy for specific requests or elements.
+In this case, take a **progressive approach**: set a protective policy like
+`strict-origin-when-cross-origin` for your website and if need be, a more permissive policy for
+specific requests or HTML elements.
 
-**Example:**
+### Example: element-level policy
 
 `index.html`:
 
-```html
-<meta name="referrer" content="strict-origin-when-cross-origin" />
-<img src="…" referrerpolicy="no-referrer-when-downgrade" />
+```html/6
+<head>
+  <!-- document-level policy: strict-origin-when-cross-origin -->
+  <meta name="referrer" content="strict-origin-when-cross-origin" />
+  <head>
+    <body>
+      <!-- policy on this <a> element: no-referrer-when-downgrade -->
+      <a src="..." href="..." referrerpolicy="no-referrer-when-downgrade"></a>
+      <body></body>
+    </body>
+  </head>
+</head>
 ```
 
+Note that Safari/WebKit may cap `document.referrer` or the `Referer` header for
+[cross-site](/same-site-same-origin/#%22same-site%22-and-%22cross-site%22) requests.
```

Comment:
Please go into that page and add a stable ID to the section. You do it like this:

```
## Same-site and cross-site {: #same-site-cross-site }
```

The ID `#same-site-cross-site` can be whatever you want.

This sends us a clue that another part of the site is linking to this section, and ensures that your link will work over the long-run (if someone edits the section heading text, your link breaks, because the current ID is generated from the text)

Reply:
```suggestion
[cross-site](/same-site-same-origin/#same-site-cross-site) requests.
```